client/js/components/procedure/Shared/SimilarStatementSubmitters/SimilarStatementSubmitters.vue

```diff
@@ -402,6 +425,15 @@ export default {
       }
     },
 
+    showUpdateForm (index) {
```

Comment:
nitpick: I think the function name is not very descriptive; I'd suggest to rename it to something like `setInitialValues` or `setFormFields`